### ISSUE TITLE
Qt implementation for NodeTree is missing

### ISSUE THREAD

**Issue by kitchoi** [enthought/pyface#809]
In order for `pyface.tree.Tree` to support Qt, we need an implementation in `pyface.ui.qt.tree.Tree`, and so on. The entire `pyface.ui.qt.tree` package is missing.

**Comment by corranwebster** [comment]
Yes, but this is way low priority - the intent is that the DataView stuff will eventually allow this to be deprecated.  It can't be yet, because TraitsUI uses this for the Wx implementation of the `TreeEditor`.
